pastas/read/menyanthes.py

```diff
@@ -24,6 +24,18 @@
 
 
 def read_meny(fname, locations=None, datatype='H'):
+    """Method to read a Menyanthes file (.men).
+
+    Parameters
+    ----------
+    fname:
+    locations
+    datatype
+
+    Returns
+    -------
+
+    """
```

Comment:
to be completed

Reply:
Actually improving the docstrings was not part of this PR but should off course be done.